dev-system/project.clj

```diff
@@ -59,9 +59,9 @@
   :repl-options {
     :init-ns user
     :timeout 300000
-    :welcome ~(do
-               (println (slurp "resources/text/banner.txt"))
-               (println (slurp "resources/text/loading.txt")))}
+    :welcome (do
```

Comment:
Out of curiosity - what did this fix?

Reply:
@ygliuvt Mentioned this to me on a pairing session we just had ... seems that this was the cause for some of the error messages (about project.clj files) we've seen when running `lein install!`.

I learned something here, though -- I though that all function calls in a project.clj file had to be escaped, since lein treats the contents of project.clj as literals (which is why you don't have to escape lists in project.clj file, e.g. you do `(:a :b)` not `'(:a :b)`).

Of course, I'll have to test this to be totally convinced the fix works :-)

Reply:
yes, what @oubiwann said.